SPECS/k3s/k3s-1.24.6.spec

```diff
@@ -1,11 +1,12 @@
 Summary:        Lightweight Kubernetes
 Name:           k3s
```

Comment:
If this remains the only spec in `k3s/` directory, then I would suggest renaming it to `k3s` and dropping the version suffix for this spec.

Reply:
I will need to add to a few more versions in future. Like k3s 1.25.4 etc. That is why maintaining the version in spec file name would be good thing here.